app/views/projects/samples/attachments/_attachment.html.erb

```diff
@@ -1,19 +1,75 @@
 <tr
   id="<%= dom_id(attachment) %>"
-  class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
+  class="
+  bg-white
+  border-b
+  dark:bg-slate-800
+  dark:border-slate-700
+  "
 >
+  <% if attachment.associated_attachment %>
+  <th class="px-6 py-4">
+    <div>
+      <div class="flex mb-4">
```

Comment:
**nitpick:** Can you add the class `items-center` here and line 26 below, this will center the icon and the text.

Reply:
Fixed in b265302acf47f8d228735e52ddb83bb1b54bfded